templates/.VERSION

```diff
@@ -1 +1 @@
-0.9.99.0915
\ No newline at end of file
+v0.9.99-212-g2c92e45
```

Comment:
@0xbaadf00d removal of `templates/.VERSION` is done by @tboerger in #74 https://github.com/go-gitea/gitea/pull/74/files#diff-1101bb4d7aabf48ba4f4b98ecd7f3efb